service/src/test/java/uk/gov/hmcts/reform/fpl/service/OrganisationServiceTest.java

```diff
@@ -57,21 +61,20 @@
 
     @BeforeEach
     void setup() {
-        LocalAuthorityUserLookupConfiguration laUserLookupConfig =
-            new LocalAuthorityUserLookupConfiguration("SA=>1,2,3");
-        organisationService = new OrganisationService(laUserLookupConfig, organisationApi,
-            authTokenGenerator, requestData);
+        lookupSpy = Mockito.spy(new LocalAuthorityUserLookupConfiguration("SA=>1,2,3"));
```

Comment:
you can try:

@ExtendWith(MockitoExtension.class)
@MockitoSettings(strictness = LENIENT)
class OrganisationServiceTest {

    @Mock
    private OrganisationApi organisationApi;

    @Mock
    private AuthTokenGenerator authTokenGenerator;

    @Mock
    private RequestData requestData;

    @Spy
    private LocalAuthorityUserLookupConfiguration lookupSpy = new LocalAuthorityUserLookupConfiguration("SA=>1,2,3");

    @InjectMocks
    private OrganisationService organisationService;

    private static final Request REQUEST = Request.create(GET, EMPTY, Map.of(), new byte[]{}, UTF_8, null);
    private static final String AUTH_TOKEN_ID = "Bearer authorisedBearer";
    private static final String SERVICE_AUTH_TOKEN_ID = "Bearer authorised service";
    private static final String USER_EMAIL = "test@test.com";
    private static final Organisation EMPTY_ORGANISATION = Organisation.builder().build();
    private static final Organisation POPULATED_ORGANISATION = buildOrganisation();

    @BeforeEach
    void setup() {
        when(authTokenGenerator.generate()).thenReturn(SERVICE_AUTH_TOKEN_ID);
        when(requestData.authorisation()).thenReturn(AUTH_TOKEN_ID);
    }

Reply:
Wonder what I did differently, last time I tried to do this the spy wasn't created properly and was trying therefore failing initialisation. 🤷 
That works so will change to that.